experimental/cloudstrapper/playbooks/roles/ansible-server/tasks/main.yaml

```diff
@@ -0,0 +1,16 @@
+#Installing Ansible Server
+#Needs to be run as sudo user (instead of becoming sudo user for each play)
+
+
+- name: installing ansible dependent packages
+  apt: 
+    name: "{{ bootPackages }}" 
+    state: present
+    update_cache: yes
+  with_items:
```

Comment:
should this contain {{bootPackages}}

Reply:
bootPackages is defined in the main.yaml file under vars. The with_items was an earlier draft that is dead code now. Removed it.